chart/open-feature-operator/README.md

```diff
@@ -84,56 +84,57 @@ The command removes all the Kubernetes components associated with the chart and
-| `flagdProxyConfiguration.port`                 | `8015`                                       | Sets the port to expose the sync API on                                           |
-| `flagdProxyConfiguration.metricsPort`          | `8016`                                       | Sets the port to expose the metrics API on                                        |
-| `flagdProxyConfiguration.image.repository`     | `ghcr.io/open-feature/flagd-proxy`      | Sets the image for the flagd-proxy deployment                                |
-| `flagdProxyConfiguration.image.tag`            | current flagd-proxy version: `v0.2.1`   | Sets the tag for the flagd-proxy deployment                                  |
-| `flagdProxyConfiguration.debugLogging`         | `false`                                      | Controls the addition of the `--debug` flag to the container startup arguments    |
-
-## Changelog
-
-See [CHANGELOG.md](https://github.com/open-feature/open-feature-operator/blob/main/CHANGELOG.md)
+| Name                                                                      | Description                                              | Value                                        |
+| ------------------------------------------------------------------------- | -------------------------------------------------------- | -------------------------------------------- |
+| `controllerManager.kubeRbacProxy.image.repository`                        | Sets the image for the kube-rbac-proxy.                  | `gcr.io/kubebuilder/kube-rbac-proxy`         |
+| `controllerManager.kubeRbacProxy.image.tag`                               | Sets the version tag for the kube-rbac-proxy.            | `v0.14.0`                                    |
+| `controllerManager.kubeRbacProxy.resources.limits.cpu`                    | Sets cpu resource limits for kube-rbac-proxy.            | `500m`                                       |
+| `controllerManager.kubeRbacProxy.resources.limits.memory`                 | Sets memory resource limits for kube-rbac-proxy.         | `128Mi`                                      |
+| `controllerManager.kubeRbacProxy.resources.requests.cpu`                  | Sets cpu resource requests for kube-rbac-proxy.          | `5m`                                         |
+| `controllerManager.kubeRbacProxy.resources.requests.memory`               | Sets memory resource requests for kube-rbac-proxy.       | `64Mi`                                       |
+| `controllerManager.manager.image.repository`                              | Sets the image for the operator.                         | `ghcr.io/open-feature/open-feature-operator` |
+| `controllerManager.manager.image.tag`                                     | Sets the version tag for the operator.                   | `v0.2.33`                                    |
```

Comment:
How will these versions be managed?

Reply:
this is generated purely from values.yaml file, so whatever version/default value is present there, it will be present in this README as well. If not, this check will fail